multipass-addressing/build.gradle

```diff
@@ -2,9 +2,12 @@ apply plugin: 'scala'
 
 version = '1.0.0'
 // Check cluster or spark environment for appropriate scala version
-def scala_version = '2.11'
-//def scala_version = '2.12'
-def spark_version = '2.4.4'
+//def scala_version = '2.11'
+def scala_version = '2.12'
+//def spark_version = '2.4.4'
```

Comment:
Please clean-up the comments which are not required.

Reply:
Thanks, Avdhesh, I cleaned up the comments which are not required.